zad3/app/models/potwor.rb

```diff
@@ -0,0 +1,2 @@
+class Potwor < ApplicationRecord
```

Comment:
Zupełnie nie było potrzeby tworzenia modelu - to dopiero na następnych zajęciach 😉 btw wszystko staramy się nazywać po angielsku 🇬🇧 🇺🇸